jwst/extract_1d/extract.py

```diff
@@ -1208,17 +1210,20 @@ def get_target_coordinates(
         else:
             targ_ra = getattr(slit, 'source_ra', None)
             targ_dec = getattr(slit, 'source_dec', None)
+            if targ_ra is None or targ_dec is None:
```

Comment:
This is necessary for NIRSpec FS mode, where the input data are in the form of SlitModels, but there's no source RA/Dec info on a per-slit basis, because there's only 1 main target. So in this case it's necessary to revert to using the generic targ_ra/targ_dec values.